ledger/src/blockstore_processor.rs

```diff
@@ -394,16 +402,16 @@ fn process_batches(
         // a push based one from the unified scheduler to the replay stage to eliminate the current
         // overhead: 1 read lock per batch in
         // `BankWithScheduler::schedule_transaction_executions()`.
-        schedule_batches_for_execution(bank, batches)
+        schedule_batches_for_execution(bank, locked_entries)
```

Comment:
please update the referred variable name in the preceding comment (and possibly word-wrapping is needed...):

```
To recite, the returned error is completely unrelated to the argument's `batches` => `locked_entries` at the
```

Reply:
08918f4a3148a3ddaa800d1009e11a3c24433cc9